app/src/main/java/example/jackliu/nonewfriends/Constants.java

```diff
@@ -0,0 +1,11 @@
+package example.jackliu.nonewfriends;
+
+/**
+ * Created by jackliu on 3/6/18.
+ */
+
+public class Constants {
```

Comment:
Are you going to add more fields to this class? or is it just for that one string?